man/ctags-optlib.7.rst.in

```diff
@@ -54,6 +54,11 @@ that of Exuberant-ctags; the newly introduced kind should be declared
 explicitly, (long) names are approved than single letter flags
 specifying kinds, and naming rules are stricter.
 
+This man page explains only stable options and flags.  Universal-ctags
+is in development, and it introduces experimental options and flags.
+They have names started with ``_``. If you are interested in them, visit
+Universal-ctags web site (https://ctags.io/).
```

Comment:
"This man page explains only stable options and flags. Universal-ctags also introduces experimental options and flags which have names starting with ``_``. For documentation on these options and flags, visit the Universal-ctags web site at https://ctags.io."

----

That reads a bit more natural to me, but at least `s/started/starting/`